scalyr_agent/agent_main.py

```diff
@@ -1750,6 +1763,12 @@ def stop(self):
         help="Read configuration from FILE",
         metavar="FILE",
     )
+    parser.add_option(
+        "--additional-config-dir",
```

Comment:
Ok, something feels a bit awkward about calling this additional-config-dir and then treating it as something extra, on top, of the normal one.  What if we just allowed for this to be a list and default it to `agent.d`.  So, we would have a config option `--config-dir` or `--config-dirs` where it could be a comma separated list of configuration directories.  Same with the environment variable.

I know, it's bit of a change from what you've done.  But, these two items are really the same type of thing and it's more general to treat it as a list.

What do you think?  I'm going back and forth on it.  If don't change it to a list, then maybe a better name. 
 The `additional-config-dir` seems just too long / awkward.  Maybe `extra-config-dir`, or `custom-config-dir`, or `secondary-config-dir`?

Ok, I'll leave it up to.

Reply:
The problem with having it as a list is that someone could redefine the list - and accidentally leave of `agent.d`, which would then break the agent (at least in k8s world where we have a bunch of essential files in `agent.d`).

Happy to rename to `extra-config-dir`?

Reply:
I also imagine the most common usage of this (k8s) will be done with environment vars, and for non-k8s usage it's not really that necessary (this feature came about because k8s configmaps can't map sub-directories to files, but no such problem exists in standalone world).